src/DigestEmail.php

```diff
@@ -0,0 +1,175 @@
+<?php
+namespace WCEmailDigest;
+use WCEmailDigest\Subscribers;
+
+class DigestEmail extends \WC_Email {
+	/**
+	 * Instance of WCEmailDigest\Subscribers
+	 * @var WCEmailDigest\Subscribers
+	 */
+	public $subscribers;
+
+	public function __construct() {
+		$this->template_base = plugin_dir_path(__DIR__) . 'templates/';
+
+		parent::__construct();
+
+		$this->schedule = $this->get_option('schedule');
+		$this->recipient = $this->get_option('recipient');
+		if( !$this->recipient ) {
+			$this->recipient = get_option('admin_email');
+		}
+	}
+
+	/**
+	 * Save admin options and maybe set CRON schedule
+	 * @return void
+	 */
+	public function process_admin_options() {
+		parent::process_admin_options();
+
+		if( $this->is_enabled() ) {
+			$this->maybeSchedule();
+		}
+		else {
+			$this->unschedule();
+		}
+	}
+
+	/**
+	 * Form fields to display in admin
+	 * @return void
+	 */
+	public function init_form_fields() {
+		$this->form_fields = [
+			'enabled' => [
+				'title' => 'Enable/Disable',
+				'type' => 'checkbox',
+				'label' => 'Enable this email digest',
+				'default' => 'no'
+			],
+			'recipient' => [
+				'title'         => 'Recipient(s)',
+				'type'          => 'text',
+				'description'   => 'Enter recipients (comma separated) for this email. Defaults to ' . get_option( 'admin_email' ),
+				'placeholder'   => '',
+				'default'       => '',
+			],
+			'schedule' => [
+				'type' => 'select',
+				'title' => 'Scheduled Frequency',
+				'description' => 'Weekly emails will be sent on Mondays. All emails are sent at 7:00am.',
+				'options' => [
+					'daily' => 'Daily',
+					'weekly' => 'Weekly'
+				],
+				'default' => 'daily'
+			],
+			'email_type' => [
+				'title'       => 'Email type',
+				'type'        => 'select',
+				'description' => 'Choose which format of email to send.',
+				'default'     => 'html',
+				'class'       => 'email_type wc-enhanced-select',
+				'options'     => $this->get_email_type_options(),
+				'desc_tip'    => true,
+			],
+		];
```

Comment:
A package like this would be awesome for testing the construction of arrays like these: 

https://github.com/spatie/phpunit-snapshot-assertions